src/data/Sort/Sort.story.tsx

```diff
@@ -16,6 +16,52 @@ export const Simple = () => {
   );
 };
 
+export const CustomIcons = () => {
+  const [dir, setDir] = useState<SortDirection>('asc');
+  return (
+    <Sort
+      direction={dir}
+      onSort={setDir}
+      neutralIcon={({ className }) => (
```

Comment:
is passing a className in like this necessary? why not just have the className on the icon you're passing?
`<svg ... className={css.neutralIconStyles} />`

Reply:
there's also `neutralIconClassName` too .. feels a bit redundant and unsure where the right place to add the className is

Reply:
Yea it is a little confusing, it wasn't my initial take on how to do it, but I just matched how `icon` and `iconClassName` work

Reply:
This would also look less confusing if I didn't inline `neutralIcon`, like this

```
const NeutralIcon = ({className}) => (
  <svg ... className={className} />
)

<Sort
  neutralIcon={NeutralIcon}
  neutralIconClassName={css.neutralIconStyles}
...
/>
```

Reply:
Ah ok I just pulled the icons into separate components and it's also much more clear in the storybook exposed code 👍 

<img width="557" alt="image" src="https://github.com/reaviz/reablocks/assets/40581813/2c91876e-11a9-4360-b027-9a647e06aa9c">